app/controllers/peer_reviews_controller.rb

```diff
@@ -31,6 +31,18 @@ def create
     end
   end
 
+  def send_zip
```

Comment:
It is not clear from the method name which zip you're trying to send. (There's the exercise template zip and the model solution zip)